File: Python/program27.py
def display(i_frequency: int):
    # Filter for negative input
    if i_frequency < 0:
        print("Invalid input")
        return

    for i_cnt in range(1, i_frequency + 1):
        print("Jay Ganesh...")

def main():
    i_count = int(input("Enter the frequency : "))
    display(i_count)

if __name__ == "__main__":
    main()

File: Java/program27.java
import java.util.Scanner;

class Program27
{
    public static void Display(int iFrequency)
    {
        int iCnt = 0;

        // Filter
        if(iFrequency < 0)
        {
            System.out.println("Invalid input");
            return;
        }

        for(iCnt = 1; iCnt <= iFrequency; iCnt++)
        {
            System.out.println("Jay Ganesh...");
        }
    }

    public static void main(String A[])
    {
        Scanner sc = new Scanner(System.in);

        int iCount = 0;

        System.out.println("Enter the frequency : ");
        iCount = sc.nextInt();

        Display(iCount);

        sc.close();
    }
}